share/spice/zomato/zomato.css

```diff
@@ -0,0 +1,9 @@
+.zci--zomato .tile--loc .tile__media__img {
+    max-width: 100% !important;
+}
+.zci--zomato .tile--loc .review-count {
```

Comment:
@dheeraj143 We'll fix this internally as it should be fixed at the template level, so other IA's won't have to adjust the CSS.